dotCMS/src/main/java/com/dotcms/publisher/pusher/PushPublisherConfig.java

```diff
@@ -134,6 +134,27 @@ public boolean justIncludesUsers() {
 		return true;
 	}
 
+	/**
+	 * Return true if the {@link PushPublisherConfig#getAssets()} list is not empty and just
+	 * contains {@link com.dotmarketing.portlets.categories.model.Category}, otherwise return false.
+	 *
+	 * @return
+	 */
+	public boolean justIncludesCategories() {
+
+		if (!UtilMethods.isSet(this.getAssets())) {
+			return false;
+		}
+
+		for (PublishQueueElement asset : this.getAssets()) {
```

Comment:
set to final

Reply:
you may reduce this to return !this.getAssets().stream.anyMatch (asset -> !"CAT".equalsIgnoreCase(asset.getAsset()))

Reply:
https://github.com/dotCMS/core/pull/23183/commits/efcc202067d1ebc9384341b6869f6b78aa350bc7#diff-28ca72a88db7305591d8dac2158e7eb1c45b87e08805666f31c0f0be5b4089cfR149